app/components/form/fields/TlsCertsField.tsx

```diff
@@ -26,8 +27,13 @@ export function TlsCertsField({ control }: { control: Control<SiloCreateFormValu
   const [showAddCert, setShowAddCert] = useState(false)
 
   const {
-    field: { value: items, onChange },
-  } = useController({ control, name: 'tlsCertificates' })
+    field: { value: items, onChange, ref },
+    fieldState: { error },
+  } = useController({
+    control,
+    name: 'tlsCertificates',
+    rules: { required: true },
```

Comment:
This seems to work, but I was surprised about it since `[]` is truthy. Found this in the [docs](https://react-hook-form.com/docs/useform/register) recommending `validate` over `required` for object and array values. I'll make that tweak now.

<img width="927" alt="image" src="https://github.com/user-attachments/assets/777756b3-28e8-4d9b-a0a3-5777db236003">